benchmarks/experimental_components/utils/timer.py

```diff
@@ -0,0 +1,98 @@
+"""Timer class based on the timeit.Timer class, but torch aware."""
+
+import timeit
+from typing import List, Optional
+
+import numpy as np
+import torch
+import utils.common as common
+
+
+__all__ = ["Timer"]
+
+
+if torch.has_cuda:
+    def timer():
+        torch.cuda.synchronize()
+        return timeit.default_timer()
+else:
+    timer = timeit.default_timer
+
+
+class Timer(object):
+    def __init__(
+        self,
+        stmt="pass",
+        setup="pass",
+        timer=timer,
+        globals: Optional[dict] = None,
+        label: Optional[str] = None,
+        sub_label: Optional[str] = None,
+        description: Optional[str] = None,
+        env: Optional[str] = None,
+        num_threads=1,
+    ):
+        if not isinstance(stmt, str):
+            raise ValueError("Currently only a `str` stmt is supported.")
+        self._stmt = stmt
+        self._setup = setup
+        self._timer = timer
```

Comment:
don't you override it below?

Reply:
Oops. This one no longer needs to be stored. Good catch.